updated configuration settings page images

fee-sah-yor opened the pull request:
## Description
This pull request introduces changes to the configuration settings page by updating the images in the documentation to match the corresponding UI changes in v7.0. The changes were made to the docs/configuration/settings.rst file.

#### Note
There are still 5 images to be updated, but I couldn't because I dont have access to those tools. I would appreciate it if someone else could address those changes in a separate issue when the PR is merged.

- the sendgrid-api-dsn image
- the queue-redis image
- the queue-beanstalkd image
- the queue-amqp image
- the social-settings image
- the smtp-dsn image

@t-education team @adiati98

<!-- PLEASE WRITE ABOVE THIS COMMENT. -->

<!--

Clearly describe what changes you have made in this PR, where our maintainers or reviewers should look at, and how to test the changes.

If the request is not complete but you want feedback or have quetions, you can select the "Draft Pull Request" option from the dropdown menu when creating the PR, then ask your questions or write your feedback in the comment.

-->

## Linked issue
Closes #452 

<!-- PLEASE WRITE ABOVE THIS COMMENT. -->

<!--

If your PR is related to a current issue, please link to that issue number. Type the keyword "Closes" followed by a hashtag (#) symbol and the issue number that you can find right after the issue title. Don't add anything else, such as period, comma, etc, to this. For example:

❌ Closes: #123.

✅ Closes #123

Doing so will automatically close the issue when one of our maintainers merges your PR.

-->

## Screenshots or screen recordings
I have attached some of the images. I changed 36 images in total
| Before | After |
|--------|-------|
| <img width="788" height="449" alt="Screenshot 2025-09-24 at 05 45 00" src="https://github.com/user-attachments/assets/16157502-53e1-49dc-827e-bb6482b39f34" /> | <img width="826" height="688" alt="Screenshot 2025-09-24 at 05 43 28" src="https://github.com/user-attachments/assets/7bc0c185-85fc-4624-a5ff-aa9c5664c4a9" /> |
| <img width="737" height="328" alt="Screenshot 2025-09-24 at 05 45 15" src="https://github.com/user-attachments/assets/6725a4a2-ee45-46d6-ab52-45074ac8a05c" /> | <img width="782" height="377" alt="Screenshot 2025-09-24 at 05 43 43" src="https://github.com/user-attachments/assets/6f2e8732-2427-4b8b-b925-db890e2cb741" />|

<!-- 

Provide screenshots or screen recordings before and after the changes, if it's a UI changes. You can simply drag and drop your files above this comment.

-->

fee-sah-yor committed:
updated configuration settings page images

adiati98 commented on docs/configuration/images/general-settings.png:
**issue:** Let's remove the active input here. The cursor triggers an active input. Users can mistaken it as a field that they need to pay attention to, while what we want here is to show how the General Settings looks like.

fee-sah-yor replied:
Alright i will work on this

adiati98 commented on docs/configuration/images/system-default-settings.png:
**issue:** I believe this one is the same as the previous one. Let's remove the highlight.

fee-sah-yor replied:
Alright i will work on this

adiati98 reviewed (changes requested):
Hey @fee-sah-yor,

Thank you so much for the PR! You did a great job. 👏🏼 

I left a couple of reviews here.
Please let me know if you have any question. 🙂

fee-sah-yor committed:
implemet requested changes

fee-sah-yor commented:
@adiati98 i have implemented your requested changes. I also updated the monitored inbox settings image to display the full image since i could not give the full screenshot previously.

adiati98 committed:
Merge branch '7.0' into update/configuration-settings-page-7.0

adiati98 reviewed (approved):
Well done and thank you for your contribution, @fee-sah-yor! ✨ 

I really appreciate your thoughtful PR description and notes, and your prompt response to all suggestions! 🫶 

![quality work GIF](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExNDBjdmVrYXU1OXdxZmo3dGxvN29nbjFtMGR2OXJobXgwcm9wOWsycyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/VhWVAa7rUtT3xKX6Cd/giphy.gif)

fee-sah-yor commented:
Thank you too @adiati98. For helping me with my debug process

adiati98 commented:
@all-contributors please add @fee-sah-yor for docs

adiati98 commented:
@all-contributors please add @adiati98 for review
